functests/performance/updating_profile.go

```diff
@@ -142,19 +142,15 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 		newNodeSelector := map[string]string{newLabel: ""}
 
 		By("Skipping test if cluster does not have another available worker node")
-		workerNodes, err := nodes.GetByRole(testclient.Client, "worker")
+
+		nonRTWorkerNodes, err := nodes.GetNonRTWorkers()
 		Expect(err).ToNot(HaveOccurred())
 
-		for _, node := range workerNodes {
-			if _, ok := node.Labels[fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT)]; ok {
-				continue
-			}
-			newWorkerNode = &node
-			break
-		}
-		if newWorkerNode == nil {
-			Skip("Skipping test because there are no additional worker nodes")
+		if len(nonRTWorkerNodes) == 0 {
+			Skip("Skipping test because there are no additional non-cnf worker nodes")
```

Comment:
maybe change non-cnf to non performance workers

Reply:
Done.